src/components/member.js

```diff
@@ -0,0 +1,30 @@
+import { members } from '../data.js';
+
+export default function Member() {
```

Comment:
Make the `Member` component only responsible for rendering 1 member. And add a new component `MemberList` that takes `members` as a prop and map it to `Member` in jsx